package.json

```diff
@@ -1,25 +1,29 @@
 {
   "name": "r7insight_docker",
-  "version": "0.9.0",
+  "version": "1.0.0",
   "description": "Forward all logs from all running Docker containers to Rapid7 InsightOps",
   "repository": {
     "type": "git",
     "url": "https://github.com/rapid7/r7insight_docker.git"
   },
+  "readme": "see [README.md](https://github.com/rapid7/r7insight_docker/blob/master/README.md)",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "make test"
   },
   "author": "Rapid7 <InsightOpsTeam@rapid7.com>",
   "license": "MIT",
   "dependencies": {
     "docker-allcontainers": "^0.4.0",
     "docker-event-log": "^0.1.2",
-    "docker-loghose": "^1.1.0",
+    "docker-loghose": "1.1.0",
```

Comment:
Why pin these versions specifically, but not the others?

Reply:
Good point, elimination process of pinning packages which break the build.

I'll explicitly pin the versions.